charts/lab-partner/templates/deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: lab-partner
+  labels:
+    {{- include "lab-partner.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
```

Comment:
It doesn't really make sense to scale a Slack bot because then it would respond multiple times to slack events. Remove the replicaCount value and hardcode this to 1. See if there is anything else we can simplify here.

Reply:
Aren't slack events sent to the bot via webhook?  with multiple replicas, only one will receive the webhook.

I don't necessarily disagree with multiple replicas, I think we could do that if we took the time to figure it out.  I'd prefer an HA deployment over a non HA one.